lib/runners/processor/fxcop.rb

```diff
@@ -0,0 +1,98 @@
+module Runners
+  # A processor class for FxCop Analyzers
+  # https://docs.microsoft.com/en-us/visualstudio/code-quality/install-fxcop-analyzers
+  # This is a beta release. We support only repositories that has .csproj file directly under analysis root directory.
+  class Processor::FxCop < Processor
+    #parameters
+    ANALYSIS_LOGFILE_PATH = '/tmp/sider_analysis_out.json'.freeze
+    RULE_ID_PATTERN = /CA[0-9]+/.freeze
+
+    Schema = StrongJSON.new do
+      let :issue, object(
+        severity: string?,
+        )
+    end
+
+    def self.ci_config_section_name
+      'fxcop'
+    end
+
+    def analyzer_name
+      'FxCop'
+    end
+
+    def analyzer_version
+      ENV['FXCOP_VERSION']
```

Comment:
Safer. If the name is not found, it raises `KeyError`.

```suggestion
      ENV.fetch('FXCOP_VERSION')
```